src/app/pages/deprecation/DeprecationAction.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace app\pages\deprecation;
+
+use Slim\Psr7\Request;
+use Slim\Views\Twig;
+use app\domain\Version;
+use app\domain\util\Redirect;
+use app\domain\Artifact;
+
+class DeprecationAction
+{
+  private Twig $view;
+
+  public function __construct(Twig $view)
+  {
+    $this->view = $view;
+  }
+
+  public function __invoke(Request $request, $response, $args)
+  {
+    return $this->view->render($response, 'deprecation/deprecation.twig', [
+      'versions' => self::getVersions(),
+      'features' => self::getFeatures()
+    ]);
+  }
+  
+  private static function getVersions(): array
+  {
+    return ["3", "4", "5", "6", "7", "8", "9.1", "9.2", "9.3"];
```

Comment:
needs to be maintained with every new release.

ReleaseInfoRepository knows all available versions. We could load all LTS versions from there and adding all versions defined in your features list to this list here.